### PR TITLE
Mention ruff as a possible linter

### PR DESCRIPTION
**Pull request by FrostyX**
See https://github.com/fedora-copr/vcs-diff-lint/pull/18

### CODE REVIEW

**Comment by FrostyX** [comment]
Do we need any more documentation within this project?
Outside of it, I will write a blogpost as we discussed.

**Review by jamacku — APPROVED** [review]
LGTM, Thank you. 

> Do we need any more documentation within this project? 
> Outside of it, I will write a blogpost as we discussed.

This should be sufficient documentation for GHA.

File: README.md [hunk 1]
```diff
@@ -82,7 +82,9 @@ Equivalent to `subdirectory` option, but multiplied. Space separated list of sub
 
 ### linter_tags
 
-Comma-separated list of linter tags (selectors) for the `vcs-diff-lint` utility (see the `--linter-tags` option). By default all linters are selected. Avaliable linter tags are `mypy`, `pylint`, and `python`.
+Comma-separated list of linter tags (selectors) for the `vcs-diff-lint` utility
```
Comment (praiskup) [inline README.md]: While on this, it is a space-separated list.

**Comment by praiskup** [comment]
Otherwise LGTM as well, feel free to merge.

**Pull request merged**